frontend/cypress/integration/featureFiles/graph_context_menu.feature

```diff
@@ -9,14 +9,12 @@ Feature: Kiali Graph page - Context menu actions
     Given user is at administrator perspective
     When user graphs "bookinfo" namespaces
 
-  @bookinfo-app
```

Comment:
Do we really want to delete this? This tag is for the demo-app detection and the test is for single cluster.

Reply:
Ah no I had some issues with this hook when running locally so I removed it but forgot to add it back.